spec/acceptance/zabbix_application_spec.rb

```diff
@@ -1,6 +1,7 @@
 require 'spec_helper_acceptance'
 require 'serverspec_type_zabbixapi'
 
+# rubocop:disable RSpec/LetBeforeExamples
```

Comment:
Did this need to be disabled?

Reply:
I assume so. Rubocop complained that this line is now useless and needs to be removed.

Reply:
ah, wrong comment. yes, rubocop said there are multiple issues for that cop. I wasn't able to fix this quickly and we really need to get this modulesync merged, so I disabled it.